File: src/main/java/com/empiricus/statusviajante/repository/CategoriaGastoRepository.java
package com.empiricus.statusviajante.repository;

import com.empiricus.statusviajante.model.CategoriaGastoModel;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

public interface CategoriaGastoRepository extends JpaRepository<CategoriaGastoModel, Long> {

    public List<CategoriaGastoModel> findAllByNomeContainingIgnoreCase (String nome);
}

File: src/main/java/com/empiricus/statusviajante/repository/CategoriaGastoRepository.kt
package com.empiricus.statusviajante.repository

import com.empiricus.statusviajante.model.CategoriaGastoModel
import org.springframework.data.jpa.repository.JpaRepository

interface CategoriaGastoRepository : JpaRepository<CategoriaGastoModel?, Long?> {
    fun findAllByNomeContainingIgnoreCase(nome: String): List<CategoriaGastoModel>
}